packages/attestation-service/package.json

```diff
@@ -55,7 +55,6 @@
     "yargs": "13.3.0"
   },
   "devDependencies": {
-    "@celo/protocol": "1.0.0",
```

Comment:
is protocol just not needed?

Reply:
Yeah, not entirely sure how it made its way in here